docs/scripts/validate_screenshot_content.py

```
@@ -0,0 +1,100 @@
+    if not png_files:
+        if args.require_files:
+            print(f"error: no PNG screenshots found in: {directory}")
+            return 1
+        print(f"no PNG screenshots found in: {directory}; skipping validation.")
+        return 0
+
+    failing = []
+    for png in png_files:
+        try:
+            mean_luma, near_black_ratio = _score_image(png)
+        except Exception as error:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)